### ISSUE TITLE
Support five facet fields

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/vt-arclight#29]
We might adjust the facet field labels but tentatively these are the fields we'd like to display as facets:

<img width="230" alt="Screen Shot 2022-10-10 at 2 45 23 PM" src="https://user-images.githubusercontent.com/101482/194957082-cf13faf4-3665-4775-83c9-52cadb6246b8.png">

where the fields reference the following columns in the metadata spreadsheet:

- Date range = `dc:date` (multi-valued)
- Language = `dc:language` (multi-valued)
- Media format = `edm:type` 
- Document type = `dc:type`
- Document format = `dc:format`

Note that the `edm:type` values in the current spreadsheet need to be cleaned up before we can use them:
- At least a few records have the value `3D`, which doesn't seem correct
- All values are currently in uppercase; we'll want to display them with init caps
- See #10 for more about this one. I think @laurensorensen might be looking into this.

**Comment by jcoyne** [comment]
Is the intention here to remove the existing "Level”, “Names”, “Place”, "Access” facets?

**Comment by laurensorensen** [comment]
We can use edm:type. I can normalize the data, it will be in physical description field in ASpace, AKA n_physdesc

I am not sure that since we are using dc:type for collection navigator that it makes sense to use it as a facet still.

**Comment by ggeisler** [comment]
Yes, because it was my understanding that we wouldn't have useful data for those facets (I expect we'll get valid data for "Level," but because we have a very shallow hierarchy, I'm not sure it will be useful, but we could wait and see when we have real data. "Access" is not particularly useful for NTA because every item will have the same value of "online".)

If that's incorrect, and you think we'll have real useful data for facets like Names and Place, let me know, because ideally those would be facets we want to offer.

**Comment by laurensorensen** [comment]
We won't have Names and Places data for this round. It would be good to have them available for future metadata additions, but that data doesn't exist for this phase.

**Comment by marlo-longley** [comment]
We created 5 individual tickets to track

Date range = dc:date (multi-valued) #141 
Language = dc:language (multi-valued) #142 
Media format = edm:type #143 
Document type = dc:type #144 
Document format = dc:format #145 

However there are open questions in this ticket around whether to use Names, Place or anything else, so I will leave this ticket open for discussion for now.

**Comment by marlo-longley** [comment]
Closing in favor of our other ticekts.
